src/dialects/abstract/query-generator.types.ts

```diff
@@ -0,0 +1,4 @@
+export interface QueryGeneratorDropSchemaQueryObject {
```

Comment:
It's used by a few more methods

```suggestion
export interface QueryWithBindParams {
```